example/client/cmd/report.go

```diff
@@ -15,17 +15,77 @@
 package cmd
 
 import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/golang/protobuf/ptypes"
```

Comment:
What version of bazel are you running locally?  Anything after 0.3.1 should be OK, as far as I can tell.